MODULE.bazel

```diff
@@ -0,0 +1,7 @@
+module(
+    name = "apple_support",
+    version = "0.11.0",
```

Comment:
so i guess we have to bump this and skylib in sync with new versions in the future?